cocos/render-scene/core/render-scene.ts

```diff
@@ -286,11 +286,11 @@ export class RenderScene {
      * @zh 从渲染场景移除所有相机
      */
     public removeCameras (): void {
-        for (const camera of this._cameras) {
+        this._cameras.forEach((camera) => {
             camera.detachFromScene();
             this._lodStateCache.removeCamera(camera);
-        }
-        this._cameras.splice(0);
+        });
+        this._cameras.length = 0;
```

Comment:
Use `this._cameras.length = 0;` instead of `this._cameras.splice(0);` to improve a little performance.